components/brave_rewards/resources/ui/reducers/rewards_reducer.ts

```diff
@@ -362,6 +362,21 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
       chrome.send('brave_rewards.disconnectWallet', [action.payload.walletType])
       break
     }
+    case types.ONLY_ANON_WALLET: {
+      chrome.send('brave_rewards.onlyAnonWallet')
+      break
+    }
+    case types.ON_ONLY_ANON_WALLET: {
+      const ui = state.ui
+
+      ui.onlyAnonWallet = !!action.payload.only
+
```

Comment:
!! here as well - so looking deliberate to me :) ignore this comment if the operator is correct.

Reply:
it's correct. see comment above